junit-platform-console/src/main/java/org/junit/platform/console/options/Theme.java

```diff
@@ -123,9 +125,8 @@ public String status(TestExecutionResult result) {
 				return aborted();
 			case FAILED:
 				return failed();
-			default:
-				return result.getStatus().name();
 		}
+		return result.getStatus().name();
```

Comment:
Unfortunately, error-prone [mandates](http://errorprone.info/bugpattern/UnnecessaryDefaultInEnumSwitch) this change, because otherwise it cannot perform its [exhaustive enum-switch analysis](http://errorprone.info/bugpattern/MissingCasesInEnumSwitch).

I suggest keeping at least "MissingCasesInEnumSwitch" switched on, because it is a [turned-on-by-default, error-level rule](http://errorprone.info/bugpatterns) (type Ctrl+F and search for "MissingCasesInEnumSwitch"), which suggests to me it's a highly mature rule and worth keeping on.